pkg/api/tag.go

```diff
@@ -197,6 +197,12 @@ func (s *server) doneSplit(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	tag.DoneSplit(tagr.Address)
+	_, err = tag.DoneSplit(tagr.Address)
+	if err != nil {
+		s.Logger.Debugf("done split: %v", err)
+		s.Logger.Error("done split: %v", err)
```

Comment:
The Error log message should not expose internals. The message should be something like this `"done split failed for address %v", tagr.Address`. Also, the Debugf would be more informative with the address in the message.